SDWebImage/NSImage+Additions.m

```diff
@@ -20,21 +23,29 @@ - (CGImageRef)CGImage {
 
 - (CGFloat)scale {
     CGFloat scale = 1;
-    CGFloat width = self.size.width;
-    if (width > 0) {
-        // Use CGImage to get pixel width, NSImageRep.pixelsWide may be double on Retina screen
-        NSUInteger pixelWidth = CGImageGetWidth(self.CGImage);
```

Comment:
This may cause unnecessary cost if the `NSImage` it's not bitmap image based. (Such as PDF image). We'd better only care about bitmap image rep.